Makefile

```diff
@@ -421,7 +421,7 @@ test-coverage: test-unit
 E2E_LABEL_FILTER = --label-filter="!openshift"
 .PHONY: test-e2e
 test-e2e: e2e-dependencies generate fmt vet ## Run e2e tests using the configured Kubernetes cluster.
-	GOFLAGS=$(GOFLAGS) USE_EXISTING_CLUSTER=true $(GINKGO) -v --trace --fail-fast $(E2E_LABEL_FILTER) ./test/e2e -- --namespace="$(NAMESPACE)" --timeout="5m" --delete-timeout="10m"
+	GOFLAGS=$(GOFLAGS) USE_EXISTING_CLUSTER=true $(GINKGO) -v --trace --fail-fast $(E2E_LABEL_FILTER) ./test/e2e  -covermode=atomic -coverprofile=coverage_e2e.out -- --namespace="$(NAMESPACE)" --timeout="5m" --delete-timeout="10m"
```

Comment:
I don't think this change is needed now? I didn't see the coverage change, at least.

Reply:
Let me remove this. Thanks Dale